### ISSUE TITLE
this kind of problem occured during the process of training 

### ISSUE THREAD

**Issue by junjieAI** [sfzhang15/RefineDet#33]
@sfzhang15 

I0119 17:40:49.416178 12387 solver.cpp:243] Iteration 380, loss = 7.68249
I0119 17:40:49.416339 12387 solver.cpp:259]     Train net output #0: arm_loss = 3.80381 (* 1 = 3.80381 loss)
I0119 17:40:49.416348 12387 solver.cpp:259]     Train net output #1: odm_loss = 4.65142 (* 1 = 4.65142 loss)
I0119 17:40:49.416353 12387 sgd_solver.cpp:138] Iteration 380, lr = 5e-05
OpenCV Error: Assertion failed ((scn == 3 || scn == 4) && (depth == CV_8U || depth == CV_32F)) in cvtColor, file /home/qi/package/software_package/opencv-3.2.0/modules/imgproc/src/color.cpp, line 9815
terminate called after throwing an instance of 'cv::Exception'
  what():  /home/qi/package/software_package/opencv-3.2.0/modules/imgproc/src/color.cpp:9815: error: (-215) (scn == 3 || scn == 4) && (depth == CV_8U || depth == CV_32F) in function cvtColor

*** Aborted at 1516354852 (unix time) try "date -d @1516354852" if you are using GNU date ***
PC: @     0x7f7e19d82428 gsignal
*** SIGABRT (@0x3e800003063) received by PID 12387 (TID 0x7f7de6e13700) from PID 12387; stack trace: ***
    @     0x7f7e19d824b0 (unknown)
    @     0x7f7e19d82428 gsignal
    @     0x7f7e19d8402a abort
    @     0x7f7e1ab9584d __gnu_cxx::__verbose_terminate_handler()
    @     0x7f7e1ab936b6 (unknown)
    @     0x7f7e1ab93701 std::terminate()
    @     0x7f7e1ab93919 __cxa_throw
    @     0x7f7e11097202 cv::error()
    @     0x7f7e11097393 cv::error()
    @     0x7f7e0f69e43e cv::cvtColor()
    @     0x7f7e1be8a86c caffe::AdjustHue()
    @     0x7f7e1be8f0ab caffe::RandomHue()
    @     0x7f7e1be8fd44 caffe::ApplyDistort()
    @     0x7f7e1bc9c3b2 caffe::DataTransformer<>::DistortImage()
    @     0x7f7e1bcf3e86 caffe::AnnotatedDataLayer<>::load_batch()
    @     0x7f7e1bd1a01f caffe::BasePrefetchingDataLayer<>::InternalThreadEntry()
    @     0x7f7e1bc91345 caffe::InternalThread::entry()
    @     0x7f7e0eaa95d5 (unknown)
    @     0x7f7e0e3576ba start_thread
    @     0x7f7e19e5382d clone
    @                0x0 (unknown)

**Comment by sfzhang15** [comment]
@junjieAI Hi，
We guess you use your own data to train the model and there are gray images in your dataset. You should set 'force_color=true'.

**Comment by junjieAI** [comment]
@sfzhang15 
Thanks for your help. maybe my own data exist gray images, and where should I set the 'force_color=true'.And If do this change, the next problem '** Aborted at 1516354852 (unix time) try "date -d @1516354852" if you are using GNU date ***' will be solved?
Hope you give me the instruction, thank you very much！

**Comment by sfzhang15** [comment]
@junjieAI 
For the python file, you should add <'focre_color': True> in 'train_transform_param'.

**Comment by junjieAI** [comment]
@sfzhang15 
yes it works, almost 2000 iterations no error was raised, but I found not only should we add  <'focre_color': True> in 'train_transform_param' for the python file , but also the train.prototxt file.
While, I use the same train data for VGG,ResNet and others, didn't occur this kind of problem（the data should not exist gray images）, by the way,thanks for you help ,and look forward to test the model.

**Comment by sfzhang15** [comment]
@junjieAI 
Adding <'focre_color': True> in 'train_transform_param' for the python file is just for adding it on the train.prototxt file.
